ersilia/default.py

```diff
@@ -104,6 +104,21 @@
         os.path.join(ROOT, "utils", "supp", _resolve_script), resolve_script
     )
 
+# Catalog table border constants
+TABLE_TOP_LEFT = "┌"
+TABLE_TOP_MIDDLE = "┬"
+TABLE_TOP_RIGHT = "┐"
+TABLE_HORIZONTAL = "─"
+TABLE_VERTICAL = "│"
+TABLE_MIDDLE_LEFT = "├"
+TABLE_MIDDLE_MIDDLE = "┼"
+TABLE_MIDDLE_RIGHT = "┤"
+TABLE_BOTTOM_LEFT = "└"
+TABLE_BOTTOM_MIDDLE = "┴"
+TABLE_BOTTOM_RIGHT = "┘"
+TABLE_CELL_PADDING = " "
+COLUMN_SEPARATOR = " | "
```

Comment:
Let's create a string enum class that has these as attributes. This way when you import it in `ersilia/hub/content/catalog.py`, there are not a lot of imports to deal with.

Reply:
@DhanshreeA I’ve implemented this in the latest commit,created a TableConstants enum class that handles all the table formatting attributes.